io_export_objex2/data_updater.py

```diff
@@ -181,8 +182,9 @@ def update_material_function(material, data, log):
     4: node_setup_simple_change_update_material_function(5), # fix version 4's update_material_function using create=False
     5: nodes_from_5, # custom cycle input sockets in 2.8x
     6: node_setup_simple_change_update_material_function(7), # add OBJEX_PrimLodFrac
+    7: node_setup_simple_change_update_material_function(8)
```

Comment:
can you add a comment on what changed in 7->8
(and yeah props to me for that upgrade system PogMe)